### PR TITLE
Fix new block map stats

### PR DESCRIPTION
**Pull request by RDruon**
See https://github.com/whamcloud/lustre-collector/pull/70

### CODE REVIEW

File: src/brw_stats.rs [hunk 1]
```diff
@@ -158,7 +158,7 @@ fn build_brw_stats(
             "dio_frags" => stats_map.get_mut_metric(DISK_IO_FRAGS),
             "discont_blocks" => stats_map.get_mut_metric(DISCONTIGUOUS_BLOCKS_TOTAL),
             "io_time" => stats_map.get_mut_metric(IO_TIME_MILLISECONDS_TOTAL),
-            _ => return,
+            _ => continue,
```
Comment (jgrund) [inline src/brw_stats.rs:161]: Shouldn't we return the new stats?

Comment (jgrund) [reply src/brw_stats.rs:161]: Something minimal for now. We can fill in detail later.

**Review by jgrund — APPROVED** [review]

**Review by jgrund — CHANGES_REQUESTED** [review]
Deb / spec / metadata files also need updating.

**Comment by RDruon** [comment]
> Deb / spec / metadata files also need updating.

Thanks, updated.

**Review by jgrund — APPROVED** [review]

**Pull request merged**